wordpress-mobile/WordPress-iOS
# Zendesk: show alert if user email is a8c

Fixes #11024 

Zendesk does not allow agents to submit tickets, and displays a `Message failed to send` error upon attempt. We can't do anything about that, so this change shows a warning when an `@automattic.com` or `@a8c.com` email address is detected:
- When the Support view is initially shown.
- When the user changes their email via `Contact Email`.

To test:
- Go to Me > Help & Support.
- If your email address is `@automattic.com`/`@a8c.com`, an alert should immediately appear.
- Select `Contact Email` and change your email to `@automattic.com`/`@a8c.com`. When `OK`, an alert should appear.

<img width="350" alt="warning" src="https://user-images.githubusercontent.com/1816888/56526698-d4c06380-6509-11e9-8835-f30e0326e076.png">

To note, you can still attempt to create a ticket using an a8c email address. It will still fail with the`Message failed to send` error. But, you've been warned. 😉
